src/content/operations/usage/_index.en.md

```diff
@@ -0,0 +1,666 @@
+  UID:                 682f791a-00b5-4f51-8249-80c7c82c4bbf
+Status:
+  Connections:
+    Endpoint:
+      Backend:          libreswan
+      cable_name:       submariner-cable-cluster3-172-17-0-10
+      cluster_id:       cluster3
+      Health Check IP:  10.3.224.0
+      Hostname:         cluster3-worker
+      nat_enabled:      false
+      private_ip:       172.17.0.10
+      public_ip:        
+      Subnets:
+        100.3.0.0/16
+        10.3.0.0/16
+    Latency:
+      Average RTT:   472623
+      Last RTT:      583181
+      Max RTT:       2603444
+      Min RTT:       80202
+      Stddev RTT:    164687
+    Status:          connected
+    Status Message:  
+  Ha Status:         active
+  Local Endpoint:
+    Backend:          libreswan
+    cable_name:       submariner-cable-cluster2-172-17-0-7
+    cluster_id:       cluster2
+    Health Check IP:  10.2.224.0
+    Hostname:         cluster2-worker
+    nat_enabled:      false
+    private_ip:       172.17.0.7
+    public_ip:        
+    Subnets:
+      100.2.0.0/16
+      10.2.0.0/16
+  Status Failure:  
+  Version:         v0.8.0-pre0-1-g5d7f163
+Events:            <none>
+```
+<!-- markdownlint-enable no-trailing-spaces -->
+To verify that service discovery is installed properly, check that the `submariner-lighthouse-coredns` service is ready:
+
+```bash
+$ kubectl -n submariner-operator get service submariner-lighthouse-coredns
+NAME                            TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE
+submariner-lighthouse-coredns   ClusterIP   100.2.177.123   <none>        53/UDP    126m
+```
+
+Next, verify that CoreDNS was properly configured to forward requests sent to `clusterset.local` to Lighthouse CoreDNS Server in the
+cluster:
+
+```bash
+$ kubectl -n kube-system describe configmap coredns
+Name:         coredns
+Namespace:    kube-system
+Labels:       <none>
+Annotations:  <none>
+
+Data
+====
+Corefile:
+----
+#lighthouse-start AUTO-GENERATED SECTION. DO NOT EDIT
+clusterset.local:53 {
+    forward . 100.2.177.123
+}
+#lighthouse-end
+.:53 {
+    errors
+    health {
+       lameduck 5s
+    }
+    ready
+    kubernetes cluster2.local in-addr.arpa ip6.arpa {
+       pods insecure
+       fallthrough in-addr.arpa ip6.arpa
+       ttl 30
+    }
+    prometheus :9153
+    forward . /etc/resolv.conf
+    cache 30
+    loop
+    reload
+    loadbalance
+}
+```
+
+Note that **100.2.177.123** is the ClusterIP address of the `submariner-lighthouse-coredns` service we verified earlier.
+
+### 2. Export Services Across Clusters
+
+At this point, we have enabled secure IP communication between the connected clusters. However, further configuration is required in order
+to signify that a Service should be visible and discoverable to other clusters in the clusterset. This is done by creating a `ServiceExport`
+object in each cluster within the namespace that the underlying Service resides in. When a `ServiceExport` is created, this will cause
```

Comment:
Add reminder that the namespace must exist in both clusters, with the default ns this is not totally clear as it automatically exists